libs/SmartStore/src/com/salesforce/androidsdk/smartstore/store/DBOpenHelper.java

```diff
@@ -103,43 +105,49 @@ public static synchronized List<String> getGlobalDatabasePrefixList(Context ctx,
 	/**
 	 * Returns the DBOpenHelper instance associated with this user account.
 	 *
-	 * @param ctx Context.
-	 * @param account User account.
+	 * @param encryptionKey Password for database.
+	 * @param ctx           Context.
+	 * @param account       User account.
 	 * @return DBOpenHelper instance.
 	 */
-	public static synchronized DBOpenHelper getOpenHelper(Context ctx,
-			UserAccount account) {
-		return getOpenHelper(ctx, account, null);
+	public static synchronized DBOpenHelper getOpenHelper(String encryptionKey, Context ctx,
+														  UserAccount account) {
```

Comment:
I assume this is just github weirdness?

Reply:
Also a side note:  If all the public API's are static synchronized this class could become a standard Kotlin object (in a future modernization PR).